base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -56,14 +56,11 @@ use tari_comms_dht::{
     domain_message::OutboundDomainMessage,
     outbound::{OutboundEncryption, OutboundMessageRequester, SendMessageResponse},
 };
-use tari_crypto::{commitment::HomomorphicCommitmentFactory, keys::SecretKey};
-use tari_p2p::{domain_message::DomainMessage, tari_message::TariMessageType};
-use tari_service_framework::{reply_channel, reply_channel::Receiver};
 #[cfg(feature = "test_harness")]
```

Comment:
why not use the cfg_if macro here as well?

Reply:
Ahh, could do but those could actually be a single use statement so just refactored them rather than adding an extra dependency.